tests/by-util/test_cp.rs

```diff
@@ -893,6 +893,28 @@ fn test_cp_preserve_no_args() {
         .succeeds();
 }
 
+#[test]
+#[cfg(unix)]
+fn test_cp_preserve_all() {
+    // TODO: create a destination that does not allow copying of xattr and context
+    new_ucmd!()
+        .arg(TEST_COPY_FROM_FOLDER_FILE)
+        .arg(TEST_HELLO_WORLD_DEST)
+        .arg("--preserve=all")
```

Comment:
this test only verifies if the command was successful. it would be better to test that some parameters are preserved!

Reply:
Hm, true. However, this PR is about ignoring the error for `all` case and not ignoring for `xattr` case. So, this one should always pass. But, I think I need to change the second test, to have one for Android, that is expected to fail, and one for other, that is expected to pass. What do you think? Otherwise, I could add some user extended attribute and for example change permissions, and check if they persist, but feels a little random :)

Reply:
nan, this would be great :)